test/dns_plugin_template_default_settings_test.rb

```diff
@@ -0,0 +1,11 @@
+require 'test_helper'
+require 'dns/dns_plugin'
```

Comment:
When I run `rake test`, this line raises:

<pre>
/home/dcleal/.rvm/gems/ruby-2.1.0@smart-proxy-plugin/bundler/gems/smart-proxy-350b4cf73f74/modules/dns/dns_plugin.rb:10:in `<class:Plugin>': uninitialized constant Proxy::Dns::ConfigurationLoader (NameError)
        from /home/dcleal/.rvm/gems/ruby-2.1.0@smart-proxy-plugin/bundler/gems/smart-proxy-350b4cf73f74/modules/dns/dns_plugin.rb:2:in `<module:Dns>'
        from /home/dcleal/.rvm/gems/ruby-2.1.0@smart-proxy-plugin/bundler/gems/smart-proxy-350b4cf73f74/modules/dns/dns_plugin.rb:1:in `<top (required)>'
        from /home/dcleal/code/foreman/smart_proxy_dns_plugin_template/test/dns_plugin_template_default_settings_test.rb:2:in `require'
</pre>